scripts/build-json/build-page-json.js

```diff
@@ -8,23 +8,15 @@ const attributes = require('./compose-attributes');
 const prose = require('./slice-prose');
 const contributors = require('./resolve-contributors');
 
-const writeToFile = (json, elementPath) => {
+function writeToFile(json, elementPath) {
```

Comment:
Thank you.

Reply:
But why? I'm not clear on what this has to do with either of the issues here

Reply:
Probably nothing but I'd be willing to accept this "violation". The project is young but once it matures we'll have to become stricter.

Reply:
You're right, it doesn't. I will put it in a separate PR if you prefer.

Reply:
Is this a style thing? I'm cool with grab bag PRs—everything's in flux right now—I just literally didn't know why you changed it from an arrow (I didn't spot `this` or anything, though I suppose I missed something). I was just curious :-)

Reply:
> Is this a style thing? 

Yes (sorry to be confusing). I'm much more familiar with using named functions than assigning anonymous functions to variables. So that's what this script does, mostly, but I copied this bit from your short-descriptions linter, and that used the other style. I though it was weird to have a mixture, and since I was editing this function anyway, thought I'd update it to be in line with the rest.

Reply:
Cool, thank you for the explanation, Will!